config/urls.py

```diff
@@ -35,6 +35,7 @@
     path("v1/webhook/", include(webhook_router.urls)),
     path("pysocial/", include("social_django.urls", namespace="social")),
     path("helauth/", include("tilavarauspalvelu.api.helauth.urls")),
+    path("palvelukartta/", include("tilavarauspalvelu.api.palvelukartta.urls")),
```

Comment:
Would this fit better as `v1/palvelukartta/` 🤔

Reply:
Unnecessary I think.

Reply:
Hmm, I guess this is reserving a top level path, which might be troublesome since we share the same paths with frontend (begin in the same domain).

Reply:
Good point.

Reply:
Changed this to `/v1/palvelukartta`